tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -20,6 +20,13 @@ USE_CHAIN_DMA=true,DEEPBUFFER_FW_DMA_MS=100"
 "cavs-sdw\;sof-mtl-rt711-4ch\;PLATFORM=mtl,NUM_DMICS=4,DMIC0_ID=2,DMIC1_ID=3,NUM_HDMIS=0,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-rt711-4ch.bin,DEEPBUFFER_FW_DMA_MS=100"
 
+# SDW + DMIC + HDMI
+"cavs-sdw\;sof-mtl-cs42l42-l0-max98363-l2-4ch\;PLATFORM=mtl,\
+NUM_SDW_AMPS=1,SDW_SPK_STREAM=SDW2-Playback,SDW_SPK_IN_STREAM=SDW2-Capture,SDW_AMP_FEEDBACK=false,\
+NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
+NUM_HDMIS=4,DEEPBUFFER_FW_DMA_MS=100,\
```

Comment:
HDMI=3, we never use 4 in practice so let's keep things simple(r).

Reply:
Ack, also noticed NUM_SDW_AMPS was changed to NUM_SDW_AMP_LINKS, will update in next version.